### PR TITLE
🚮  amp-carousel 0.1: cleanup inactive experiment

### PR DESCRIPTION
**Pull request by samouri**
Removes inactive experiment

### CODE REVIEW

**Comment by samouri** [comment]
Just noticed this modification is quite correct yet. Should only use scroll snap points for iOS except v13

**Review by rcebulko — APPROVED** [review]
Approving to unblock provided you double-check the conditional is right

File: extensions/amp-carousel/0.1/slidescroll.js [hunk 1]
```diff
@@ -143,20 +142,10 @@ export class AmpSlideScroll extends AMP.BaseElement {
...
     /** @private {boolean} */
-    this.shouldDisableCssSnap_ = Services.platformFor(this.win)
-      .getIosVersionString()
-      .startsWith('10.3')
-      ? true
-      : this.isIos_
-      ? false
-      : !isExperimentOn(this.win, 'amp-carousel-chrome-scroll-snap');
+    this.shouldDisableCssSnap_ =
+      !this.isIos_ || platform.getIosVersionString().startsWith('10.3');
```
[the remark this answers is not in the document]
Comment (samouri) [reply extensions/amp-carousel/0.1/slidescroll.js:148]: Feel free to block if you are unsure about the conditional. This is super low priority, just noticed it while doing unrelated things in `amp-carousel`.

**Comment by samouri** [comment]
Adding @alanorozco as a reviewer because I realize @rcebulko isn't an owner 🙃

**Pull request closed without merging**